alacritty_terminal/src/term/mod.rs

```diff
@@ -1586,7 +1586,10 @@ impl<T: EventListener> Handler for Term<T> {
 
     #[inline]
     fn move_forward_tabs(&mut self, count: u16) {
-        trace!("[unimplemented] Moving forward {} tabs", count);
+        trace!("Moving forward {} tabs", count);
+
+        self.put_tab(count);
+        self.damage_cursor();
```

Comment:
you should damage the way I've suggest in the patch.

Reply:
Shouldn't this be placed in `fn put_tab` instead?

Reply:
my patch put it there?

Reply:
yes